sky/tools/sky_snapshot/main.cc

```diff
@@ -20,6 +20,7 @@
 
 void Usage() {
   std::cerr << "Usage: sky_snapshot"
+            << " --" << switches::kPackages
             << " --" << switches::kPackageRoot << " --" << switches::kSnapshot
```

Comment:
Ok.  This usage line is really vague.  For example, it thinks switches::kDepfile is required too and doesn't say what any of these arguments should be.  I'll clean it up a bit.